packager/mpd/base/mpd_utils.cc

```diff
@@ -18,6 +18,7 @@
 #include "packager/mpd/base/content_protection_element.h"
 #include "packager/mpd/base/representation.h"
 #include "packager/mpd/base/xml/scoped_xml_ptr.h"
+#include "packager/media/base/protection_system_specific_info.h"
```

Comment:
Move it after line 16 (to keep it sorted)

Reply:
I just was thinking what is wrong with includes sections %)
Done.